pkg/block/blocktest/adapter.go

```diff
@@ -296,7 +311,8 @@ func testAdapterGetRange(t *testing.T, adapter block.Adapter, storageNamespace s
 		{"read_prefix", 0, len(part1) - 1, part1, false},
 		{"read_middle", 8, len(part1) + 6, "the first part this is", false},
 		// {"end_smaller_than_start", 10, 1, "", false},  // TODO (niro): To be determined
-		{"negative_position", -1, len(part1), "", true},
+		//{"negative_position", -1, len(part1), "", true}, // S3 and Azure not aligned
```

Comment:
so I assume that the block adapter should not block and the s3 gateway should align

Reply:
Not clear - since this is a contractual test, and the block adapters don't align any behavior is acceptable